ddtrace/tracer/tracer.go

```diff
@@ -206,9 +211,36 @@ func Start(opts ...StartOption) {
 		logStartup(t)
 	}
 
+	// store the configuration in an in-memory file, allowing it to be read to
+	// determine if the process is instrumented with a tracer and to retrive
+	// relevant tracing information.
+	storeConfig(t.config)
```

Comment:
👋 I came across the code in this PR (and the RFC for it) today. It's really cool. I didn't know about the memfd syscalls - I might have some use cases for it in the future as well!

Anyway, I'm commenting b/c I think there is a slight problem here. This code is only considering the tracer product. I think there is a problem when a customer is only enabling the profiler, but not the tracer. This does happen sometimes in the real world AFAIK.